app/tests/api/test_access_logging_pii.py

```
@@ -0,0 +1,105 @@
+    captured = capsys.readouterr()
+    log_output = captured.out + captured.err
+    
+    # Should have logged the warning with count
+    assert "resource_not_found" in log_output
+    assert "missing_user_count=1" in log_output
+    
+    # Should NOT contain the raw user ID
+    assert "missing-user-999" not in log_output
+
+
+def test_add_member_exception_no_raw_ids_in_logs(client, vault_repo, user_repo, capsys):
```

Ruff (0.15.1)

[warning] Unused function argument: `vault_repo`

(ARG001)